llpc/test/shaderdb/relocatable_shaders/PipelineVsPs_PsInput.pipe

```diff
@@ -1,6 +1,6 @@
 ; BEGIN_SHADERTEST
 ; RUN: amdllpc -spvgen-dir=%spvgendir% -enable-relocatable-shader-elf -o %t.elf %gfxip %s -v | FileCheck -check-prefix=SHADERTEST %s
-; SHADERTEST: SPI_PS_INPUT_CNTL_0                           0x0000000000000000
+; SHADERTEST: SPI_PS_INPUT_CNTL_0                           0x0000000000000020
```

Comment:
You might want to write more tests for this.  This test is not exercise all of the code path you have because the compilation will only have unlinked shaders.  You might want to test a full pipeline compilation with this same pipeline.

Also, the pipelines in this directory only run on gfx9, so you are not testing the gfx6 config builder code with this test.

Reply:
Yes, I should have written more tests.